diff -urN epson-laser-printer-escpage.orig/src/filter.c epson-laser-printer-escpage/src/filter.c
--- epson-laser-printer-escpage.orig/src/filter.c	2013-02-05 15:05:36.769014015 +0900
+++ epson-laser-printer-escpage/src/filter.c	2013-02-05 15:07:13.025014004 +0900
@@ -39,6 +39,7 @@
 #include <errno.h>
 #include <cups/cups.h>
 #include <cups/raster.h>
+#include <signal.h>
 
 #include "err.h"
 #include "mem.h"
@@ -619,7 +620,20 @@
   cups_raster_t *ras; /* raster stream for printing */
   cups_page_header_t header; /* page device dictionary header */
   EPS_JOB_ATTRIB fopt;
-  
+
+#if defined(HAVE_SIGACTION) && !defined(HAVE_SIGSET)
+    struct sigaction action;      /*   Actions for POSIX signals */
+#endif /*   HAVE_SIGACTION && !HAVE_SIGSET */
+    #ifdef HAVE_SIGSET
+    sigset(SIGPIPE, SIG_IGN);
+#elif defined(HAVE_SIGACTION)
+    memset(&action, 0, sizeof(action));
+    action.sa_handler = SIG_IGN;
+    sigaction(SIGPIPE, &action, NULL);
+#else
+    signal(SIGPIPE, SIG_IGN);
+#endif /*   HAVE_SIGSET */
+
   debug_msg("%s:%d \t<<%s>>: \t\tStart()\n", __FILE__, __LINE__, __FUNCTION__);
 
 /* attach point */
